packages/app/src/components-styled/line-chart/index.tsx

```diff
@@ -356,7 +356,9 @@ function formatDefaultTooltip<T extends Value>(
   if (isDaily) {
     return (
       <>
-        <Text as="span" fontWeight="bold">{formatDateFromMilliseconds(value.__date.getTime())}: </Text>
+        <Text as="span" fontWeight="bold">
+          {formatDateFromMilliseconds(value.__date.getTime())}:{' '}
```

Comment:
nit-nitpick; When combining strings I usually prefer backticks with interpolation.

```suggestion
          {`${formatDateFromMilliseconds(value.__date.getTime())}: `}
```